lib/class-wp-rest-reusable-blocks-controller.php

```diff
@@ -0,0 +1,353 @@
+<?php
+/**
+ * Reusable Blocks REST API: WP_REST_Reusable_Blocks_Controller class
+ *
+ * @package gutenberg
+ * @since 0.10.0
+ */
+
+/**
+ * Controller which provides a REST endpoint for Gutenberg to read, create and edit reusable blocks. Reusable blocks are
+ * stored as posts with a custom post type.
+ *
+ * @since 0.10.0
+ *
+ * @see WP_REST_Controller
+ */
+class WP_REST_Reusable_Blocks_Controller extends WP_REST_Controller {
```

Comment:
A reusable block is saved as just a post, yeah? Could we be leveraging more of the existing posts endpoint? The endpoint itself? `extends WP_REST_Posts_Controller`?